Tasks/ServiceFabricDeploy/task.json

```diff
@@ -16,8 +16,11 @@
     "version": {
         "Major": 1,
         "Minor": 3,
-        "Patch": 5
+        "Patch": 6
     },
+    "demands": [
+        "Cmd"
```

Comment:
Short answer: with where we're at today, cmd suffices.

The dependency is really an agent capable that supports the "PowerShell3" task execution handler.